vector-im/element-android
# [Poll] History list: details screen of a poll (PSG-1041, PSG-1151)

## Type of change


## Content


Adding last missing screen for poll history feature: a screen showing details of a poll when tapping on an item from the list.
Voting is enabled from the screen when poll is still active.

_Note: this PR also enables the poll history feature in release mode._

## Motivation and context


Closes #8056 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/216567804-78f0ad65-4c9d-48d7-b575-a9292a9bb093.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/216567817-dd7c6bc7-c4d6-49bd-9d53-3afada990d56.png" width=300 />

<img src="https://user-images.githubusercontent.com/46314705/216567946-e42a8a90-edb7-47f2-980e-99fa6b2690dc.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/216567970-00fa1fae-838a-4a86-8376-dd75ab39a0c4.png" width=300 />

## Tests



- Open the profile screen of a room
- Open the poll history screen
- Press an active poll from the list
- Check the details screen is opened with correct content
- Check you can vote to the poll
- Go back to the poll history list
- Press an ended poll from the list
- Check the details screen is opened with correct content
- Press the go to timeline button
- Check the timeline is opened around the end poll event
- Press back
- Check it opens the room list

## Tested devices

- OS version(s): Android 11

## Checklist
